app/src/main/java/org/wikiedufoundation/wikiedudashboard/common/BindingAdapter.kt

```diff
@@ -0,0 +1,36 @@
+package org.wikiedufoundation.wikiedudashboard.common
+
+import android.annotation.SuppressLint
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.databinding.BindingAdapter
+import com.bumptech.glide.Glide
+import org.wikiedufoundation.wikiedudashboard.R
+import java.text.SimpleDateFormat
+import java.util.*
+
+/**
+ * A simple [KotlinFile].
+ * It's responsible for custom bindings that are not allowed by default in Android Studio
+ */
+
+
+/**
+ * Use Kotlin [ExtensionFunctionType] to load the image with [BindingAdapter]
+ ***/
+@BindingAdapter("loadImageUrl")
+fun ImageView.setGlideImage(thumbUrl: String?) {
+    Glide.with(this)
+            .load(thumbUrl)
+            .into(this)
+}
+
+/**
+ * Use Kotlin [ExtensionFunctionType] to load format and show the date with [BindingAdapter]
+ ***/
+@SuppressLint("SimpleDateFormat")
+@BindingAdapter("dateToString")
+fun TextView.setDate(date: Date?) {
+    val dateFormat = SimpleDateFormat(this.context.getString(R.string.time_format_12_hours))
+    date?.let { this.text = dateFormat.format(it).toString() }
+}
```

Comment:
Can we move these 2 functions in ViewUtils.kt file. We already have other extension functions there and we will only need 1 file to maintain.

Reply:
It's possible to move, of course. Although these are extension functions with binding purpose, only for layouts. Are you sure about this changes?